data/Harshitahusts.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "Harshit Gupta",
+  "displayStatsPublic": ture,
+  "avatar": "https://github.com/Harshitahusts.png",
+  "bio": "**Open Source** | Java Developer ",
+  "tags": [
+    "Open Source",
+    "JAVA",
+    "MERN",
+    "SPRING",
+    "Docker",
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/Harshit54283"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/Harshitahusts"
+    }
+    
+  ],
+  
+  "links": [
+    {
+      "group": "Socials",
+      "name": "Twitter"
```

```suggestion
      "name": "Twitter",
```
Please add comma here